File: Frontend/src/classes/DiagnovirTester.ts
import User from './User'

export default class DiagnovirTester extends User {
  
  public constructor(name: string, mail: string, password: string, role: string, address: string, phoneNum: string, hesCode: string) {
    super(name, mail, password, role, address, phoneNum, hesCode);
  }
}

File: Backend/src/main/java/com/example/backend/DiagnovirTester.java
package com.example.backend;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class DiagnovirTester extends User{

    //Constructor
    public DiagnovirTester(String name, String mail, String password, String role, String address, String phoneNum, String hesCode){
        super(name, mail, password, role, address, phoneNum, hesCode);
    }

}
